### DISCUSSION TITLE
Output file with non-truncated swissprot coment line

### DISCUSSION THREAD

**Discussion by tenhaken** [bbuchfink/diamond#594] [Q&A]
We use Diamond for basic annotation of trinity output files, using the swissprot database. I there an output option availabe, that will not truncate the full comment / annotation line from swissprot?
Comments are welcome

**Comment by bbuchfink** [comment]
You can use the `-f` option to customize the output format, then use the `stitle` or `salltitles` output fields, see here: https://github.com/bbuchfink/diamond/wiki/3.-Command-line-options#output-options
